### PR TITLE
[Homework #3] - HTML by Santiago Pereira

### PR DESCRIPTION
**Pull request by Santiago-Pereira**

### CODE REVIEW

**Review by ilya-ischenko — COMMENTED** [review]
Looks good to me!

1. Do not you want to use img tag for svg icons? Your code hard to read because icons use a lot of place in your code
2. Social media icons should be clickable, so they should be links
3. Rename your homework folder to lower-case, "Santiago-Pereira" -> "santiago-pereira" to keep consistency

File: homeworks/santiago-pereira/css-lecture-04/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (ilya-ischenko) [inline homeworks/santiago-pereira/css-lecture-04/index.html:11]: I see that you are trying to push file related to homework-4 in this PR
1Pr = 1 homework
Delete files related to hw4, please

**Comment by Santiago-Pereira** [comment]
hi, I deleted homework 4 folder's , I hope everything is ok.
thank you!

**Review by ilya-ischenko — APPROVED** [review]
Well done!
Approved.

**Pull request merged**